index.html

```diff
@@ -0,0 +1,69 @@
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="index.css">
+  </head>
+  <body>
+    <form action="contactForm.php" method="POST" class="contact-form">
+
+  <div class="cf-item">
+    <label for="fname">First Name</label>
+    <input class="cf-input" type="text" id="fname" name="firstname" placeholder="Your name..">
```

Comment:
The input font is a bit too small, and therefore not very accessible. iOS requires a minimum size of 16px otherwise it zooms in on the input element which can be annoying.